hostinit/setup.go

```diff
@@ -725,5 +722,10 @@ func (init *HostInit) fetchRemoteTaskData(ctx context.Context, taskId, cliPath,
 	var cancel context.CancelFunc
 	ctx, cancel = context.WithTimeout(ctx, 15*time.Minute)
 	defer cancel()
-	return errors.WithStack(makeShellCmd.Run(ctx))
+
+	if err := makeShellCmd.Run(ctx); err != nil {
```

Comment:
I added this to log errors when the remote command fails, but this seems like it should be done in subprocess/remote_command.go. @tychoish thoughts?